### ISSUE TITLE
Color-contrast issue from disabled toggle label.

### ISSUE THREAD

**Issue by mslonli** [OfficeDev/office-ui-fabric-react#7612]
How to repro: The 'Off' label has color-contrast issue.

`<Toggle defaultChecked={false} disabled={true} label="Disabled and unchecked" onText="On" offText="Off" onFocus={() => console.log('onFocus called')} onBlur={() => console.log('onBlur called')} />`

**Comment by betrue-final-final** [comment]
Disabled controls do not need to pass WCAG contrast. Here's the excerpt:

"The visual presentation of text and images of text has a contrast ratio of at least 4.5:1, except for the following:

Large Text
Large-scale text and images of large-scale text have a contrast ratio of at least 3:1;

Incidental
Text or images of text that are part of an **inactive user interface component**, that are pure decoration, that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.

**Comment by iamrafan** [comment]
@betrue-final-final But the text "off" is still available to the AT users and hence shouldn't it be considered as not being part of the disabled control?

**Comment by dzearing** [comment]
Paging @betrue-final-final

**Comment by betrue-final-final** [comment]
I like paging MUCH better than pinging. It's more human and less submarine.

Anyway about the Off/On label. It's true that blind users can use scan mode to navigate to a disabled control, but tab or control navigation skips the entire control. It would also be odd for a random "Off/On" label to be more visible than the rest of the control.

**Comment by Vitalius1** [comment]
@betrue-final-final what is the verdict? Are we having an issue to fix here or everything is ok with the disabled toggle on/off text in HC mode? Would appreciate your input so I can close the issue either by fixing or mark it as a non-issue. Thanks!

**Comment by betrue-final-final** [comment]
I thought I answered the question. The entire disabled control does not need to pass contrast. In high contrast though, these things are customarily all rendered using "graytext" color, which is yellow in HC Black.

**Comment by Vitalius1** [comment]
Closing this issue and marking as a non-issue. (@betrue-final-final to confirm).
Currently, if the `Toggle` is disabled and using the regular tab navigation in HC the On/Off label matches the color of the `pill` and `thumb`.  As @betrue-final-final mentioned it would be odd to have it differentiate from the rest of the control.
Also, hiding it from AT like Narrator's scan mode is not possible as it requires detecting the mode so we can conditionally render the label.

@mslonli If there is something you think we are misunderstanding, feel free to comment and we will investigate further. Thanks!

**Comment by iamrafan** [comment]
@Vitalius1 I do not think there is a need to detect the mode always, the label text "off" can be always hidden from the ATs using aria-hidden=true.

**Comment by iamrafan** [comment]
Related: https://github.com/w3c/wcag/issues/567
